app/scripts/views/decorators/progress_indicator.js

```diff
@@ -25,26 +25,32 @@ define(function (require, exports, module) {
     return deferred.promise;
   }
 
-  function showProgressIndicator(handler, _el, delayMills) {
-    var el = _el || 'button[type=submit]';
-    const delayHandlerByMills = delayMills || 0;
+  function showProgressIndicator(handler, el = 'button[type=submit]', delayHandlerByMills = 0) {
+    return function (...args) {
+      const target = this.$(el);
+      const minProgressIndicatorMs = parseInt(target.data('minProgressIndicatorMs') || 0, 10);
```

Comment:
nit: I figured it was the radix but had to look it up to be sure. Maybe make a constant to save future self the lookup? Possibly `RADIX`?